WcaOnRails/config/locales/de.yml

```diff
@@ -2244,6 +2261,7 @@ de:
       must_register: muss sich für mindestens eine Disziplin anmelden
       #original_hash: abdc718
       exceeds_event_limit:
+        zero: zero
```

Comment:
typo / miss?

Reply:
Hmm, I think it's actually neither typo nor miss, let me explain.
There is nothing to translate, like, there exists nothing in the en.yml version (zero does not exist at all, see
https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/config/locales/en.yml#L1128-L1130)
This does make sense to me, extending the logic to zero this would have to be "you must register in zero events". Do such competitions exist where (for certain registrants) not a single event registration is allowed?
And because Internationalize complains otherwise, I just put a (hopefully harmless) dummy to fill the empty field. Does the zero case exist in some language with a meaningful wording, or is it even displayed?